### PR TITLE
Add docs for how to show and hide toasts programmatically.

### PR DESCRIPTION
**Pull request by hnrchrdl**
# Description

Read #428 and added some docs on how to show and hide a toast from programmatic context.

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have not generated a new package version. (the maintainers will handle that)

### CODE REVIEW

File: examples/ToastProgrammatically.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Button from '../src/Button'
+
+function() {
+    window.Materialize.toast('I am a toast!', 4000);
+}
```
Comment (alextrastero) [inline examples/ToastProgrammatically.js]: what is the purpose of this function?

File: examples/ToastProgrammatically.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+<div>
+    <Button onClick={() => {
+            window.Materialize.toast('I am a toast!', 50000);
+        }
+    }>
+        Click to show
+    </Button>
+    <Button onClick={() => {
+            const toast = document.querySelector('#toast-container>.toast');
+            console.log(toast)
```
Comment (alextrastero) [inline examples/ToastProgrammatically.js]: leftover?

Comment (hnrchrdl) [reply examples/ToastProgrammatically.js]: removed.

File: examples/ToastProgrammatically.js (hunk 1 above)
Comment (hnrchrdl) [reply examples/ToastProgrammatically.js]: removed.

**Comment by hnrchrdl** [comment]
Thanks for reviewing and sorry for insufficiencies of my initial commits. Should be fixed now.
I will check the new beta when I find the time. Is it supposed to be mostly compatible with older versions? Glad to hear that JQuery is going to be dropped from this.

**Comment by alextrastero** [comment]
Thanks for the updates @hnrchrdl.

As for the beta, it's in really early stages of development, it's a message to get users to write issues from real projects, thanks!

**Comment by JazzBrown1** [comment]
Bit hacky but this works for me, for now:

Stuck it in a string and eval so my webpacks would compile it...... There are defo better ways of doing this...

  const showToast = (text) => {
    const string = `typeof M !== 'undefined' ? M.toast({html: '${text'}'}) : null;`;
    return eval(string);
  };

**Pull request merged**